jobclient/src/main/java/com/twosigma/cook/jobclient/Instance.java

```diff
@@ -220,6 +244,14 @@ public Status getStatus() {
             return _status;
         }
 
+        public Integer getProgress() {
```

Comment:
Can we add document something like "return null if progress info is not available etc. "

Reply:
Done.